browser/ntp_background/ntp_p3a_helper_impl.h

```diff
@@ -48,25 +56,30 @@ class NTPP3AHelperImpl : public NTPP3AHelper {
                          bool is_constellation);
 
  private:
-  std::string BuildHistogramName(const std::string& creative_instance_id,
-                                 const std::string& event_type);
-
-  void RecordMetric(const std::string& histogram_name,
-                    int count,
-                    bool is_constellation);
+  void RecordCreativeMetric(const std::string& histogram_name,
+                            int count,
+                            bool is_constellation);
   void RemoveMetricIfInstanceDoesNotExist(
       const std::string& histogram_name,
       const std::string& event_type,
       const std::string& creative_instance_id);
+  void CleanOldCampaignsAndCreatives();
 
   void UpdateMetricCount(const std::string& creative_instance_id,
                          const std::string& event_type);
 
+  void UpdateCampaignMetric(const std::string& campaign_id,
+                            const std::string& event_type);
+
   void OnLandingStartCheck(const std::string& creative_instance_id);
 
   void OnLandingEndCheck(const std::string& creative_instance_id,
                          const std::string& expected_hostname);
 
+  void OnUpdated(NTPBackgroundImagesData* data) override;
```

Comment:
chromium code usually adds a comment before block of overridden methods which specifies which class methods are overridden:
`// NTPBackgroundImagesService::Observer:`

Reply:
see https://github.com/brave/brave-core/blob/master/components/ntp_background_images/browser/view_counter_service.h#L148